views/localweb.html

```diff
@@ -0,0 +1,480 @@
+                    <div><div class="author">{{ gettext("Quote 3 Author") }}</div></div>
+                  </div>
+                </div>
+                <ol class="carousel-indicators">
+                  <li data-target="#quote-carousel" data-slide-to="0" class="quote-container active"></li>
+                  <li data-target="#quote-carousel" data-slide-to="1" class="quote-container"></li>
+                  <li data-target="#quote-carousel" data-slide-to="2" class="quote-container"></li>
+                </ol>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+    </div>
+
+
+
+
+
+  </main>
+
+  <footer class="footer">
+
+    {% block footer %}
+    <div id="footer-actions">
+      {% if layout %}
+      <a class="load-more">{{ gettext("Load more") }}
+        <span class="icon-angle-down"></span></a>
+      {% endif %}
+    </div>
+    {% endblock %}
+
+    <div class="partnerWrapper">
+      <nav class="navbar-footer">
+        <div class="container">
+          <ul>
+            <li><a href="/{{localeInfo.lang}}/about">{{ gettext('About') }}</a></li>
+            <li><a href="/{{localeInfo.lang}}/mentor">{{ gettext('Teach') }}</a></li>
+            <li>
+              <a href="https://sendto.mozilla.org/page/contribute/join-mozilla?source=join_link">
+                {{ gettext('Donate') }}
+              </a>
+            </li>
+            <li><a href="mailto:help@webmaker.org">{{ gettext('Contact') }}</a></li>
+            <li><a href="/{{localeInfo.lang}}/privacy">{{ gettext('Privacy') }}</a></li>
+            <li><a href="/{{localeInfo.lang}}/terms">{{ gettext('Legal') }}</a></li>
+            <li>
+              <a href="https://support.mozilla.org/products/webmaker">
+                {{ gettext('Help') }}
+              </a>
+            </li>
+          </ul>
+          <div class="moz-logo">
+            <a href="https://mozilla.org/">
+              <img src="/img/logo-mozilla-white.png" alt="mozilla logo" height="41" width="100"/>
+            </a>
+          </div>
+        </div>
+      </nav>
+    </div>
+  </footer>
+</div>
+
+<script src="https://mozorg.cdn.mozilla.net/tabzilla/tabzilla.js"></script>
+<script src="/bower_components/jquery/jquery.min.js"></script>
+<script src="/bower_components/bootstrap/js/transition.js"></script>
+<script src="/bower_components/magnific-popup/dist/jquery.magnific-popup.min.js"></script>
+<script src="/bower_components/bootstrap/js/carousel.js"></script>
+
+{% block scripts %}
+<script>
+  $window = $(window);
+  //parallax
+  $(window).scroll(function() {
+    var yPos = -($window.scrollTop() / 50);
+    $("#moi-video").css({ backgroundPosition: "50% "+ yPos + "px" });
+  });
+  // video lightbox
+  $(".moi-video-link").magnificPopup({
+    type: "iframe"
+  });
+  // quote carousel
+  $("#quote-carousel").carousel();
+</script>
+
+{% endblock %}
+
+{% if ga_account %}
```

Comment:
Would like to double check if this is how I include the GA tracking code that webmaker.org is using